File: 2.Implement/Delivery_Server/src/main/java/com/delivery/app/Delivery/data/dto/request/market/MarketSignUpDto.java
package com.delivery.app.Delivery.data.dto.request.market;

import lombok.Getter;
import lombok.NoArgsConstructor;

@Getter
@NoArgsConstructor
public class MarketSignUpDto {
    private String name;
    private String loginId;
    private String loginPwd;
    private String phoneNumber;
    private Double longitude;
    private Double latitude;
    private String description;
    private String address;

    public boolean hasNull() {
        return   name == null
                || loginId == null
                || loginPwd == null
                || phoneNumber == null
                || longitude == null
                || latitude == null
                || description == null
                || address == null;
    }
}

File: 2.Implement/DeliveryApp/app/src/main/java/com/project/deliveryapp/retrofit/dto/request/market/MarketSignUpDto.kt
package com.project.deliveryapp.retrofit.dto.request.market

import android.os.Parcelable
import kotlinx.parcelize.Parcelize

@Parcelize
data class MarketSignUpDto(
    val name: String,
    val loginId: String,
    val loginPwd: String,
    val phoneNumber: String,
    val latitude: Double,
    val longitude: Double,
    val description: String,
    val address: String,
): Parcelable {
    fun isOkay(): Boolean {
        return     name.isNotBlank()
                && loginId.isNotBlank()
                && loginPwd.isNotBlank()
                && phoneNumber.isNotBlank()
                && address.isNotBlank()
    }
}
